src/test/java/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,180 @@
+package hw2.ex1;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Exercise1 {
```

Comment:
Why do you decide that **Exercise1` should contain several tests instead of one?

Reply:
This requirement in the task "4. Test methods should have a name in accordance with checking functionality" made me think that I should create several tests. 
Do I need to redo them into one test method?

Reply:
This was a test scenarios with the steps which should be automated. It should be one tets